content/docs/concepts/traffic-management/index.md

```diff
@@ -194,6 +194,59 @@ Services can actively shed load by responding with an HTTP 503 to a health
 check. In such an event, the service instance will be immediately removed
 from the caller's load balancing pool.
 
+## Locality Load Balancing
+
+{{< warning >}}
+This feature is currently experimental. To enabled it, set the `PILOT_ENABLE_LOCALITY_LOAD_BALANCING` environment variable in all Pilot instances.
+{{< /warning >}}
```

Comment:
This is not the right place for this kind of information. The concepts doc doesn't just talk about Pilot. It talks about what Istio can do in general, agnostic of platforms.. So, to add locality load balancing here, you simply add a subsection and talk about how users get out of box locality load balancing based on geographic regions, etc. (omit all flags stuff/experimental stuff/etc.). 

Then in a task (locality based routing), you specify that people need to enable it explicitly for 1.1 and that they should annotate the nodes, or add locality to their service entries. You don't need the usual verify if it works, blah.. 

cc @frankbu

Reply:
Another alternative is to move this into the operations guide - add an entry `Locality Load Balacancing` with this content [here](https://preliminary.istio.io/help/ops/traffic-management/). Then in the `Load Balancing` concepts section just add a single sentence or two with a reference up to the operations guide.